composefiles/swarm-uniflow.yml

```diff
@@ -302,7 +302,43 @@ services:
         limits:
           cpus: ${UR_RES_LIMIT_CPUS}
           memory: ${UR_RES_LIMIT_MEM}
-      
+
+  inventory:
+    image: frinx/frinx-inventory-server:latest
+    logging:
+      driver: "json-file"
+      options:
+        max-file: "3"
+        max-size: "10m"
+    environment:
+      - DATABASE_URL=postgres://postgres:postgres@uniconfig-postgres:5432/inventory?sslmode=disable
+      - UNICONFIG_API_URL=https://uniconfig:8181/rests
```

Comment:
how does this work with multiple zones ?

also, isn't that value "uniconfig" in some env var ? and thus should be reused here ?

Reply:
Uniconfig env is not stored in the .env file. We are using ./generate_uc_compose.sh where service name (URL name) is explicitly generated. Also, the current version of the inventory server is not prepared for multi-zone deployment. Thus this URL is hardcoded just for single-node deployment at this moment.